docs/experiment/guides/troubleshooting/sample-ratio-mismatch.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Troubleshooting Sample Ratio Mismatch (SRM)
+description: What to do if you have a Sample Ratio Mismatch (SRM)
+---
+
+<style type="text/css" rel="stylesheet">
+h4 {
+  display:inline
+}
+</style>
+
+!!! info
+    This doc already assumes that you know what an SRM is. If you do not know what an SRM is, you should read this [Help Center Article](https://help.amplitude.com/hc/en-us/articles/8043418569371-Sample-ratio-mismatches-Debug-issues-with-experiment-allocations). 
+
+This guide provides a deeper dive into the process of troubleshooting and debugging a sample ratio mismatch (SRM). This guide assumes that you're using the "end-to-end" Experiment product. However, you may apply some of the same debugging steps to experiments which have been set up in Experiment Results.
+
+!!! info
+    This list is sorted in rough order of what the possible cause is. 
+
+#### 1. Are you using a custom exposure event or the Amplitude exposure event?
+<p style="display:inline;">Amplitude recommends you use the <a href="../../../general/exposure-tracking">Amplitude exposure event</a>. When using client-side SDKs the Amplitude exposure event is automatically tracked when the variant is accessed from the cache, not when variants are fetched. When using local evaluation, there is no assignment event and requires the customer to use a custom exposure event. If you use a custom exposure event make sure you send it when the user experiences the variant. More importantly, the custom exposure event may happen before assignment, meaning that the user property is not yet set, and the initial custom exposure event is not actually counted as an exposure in analysis. Note that it does not cost extra money to use Amplitude Exposure events.</p>
```

Comment:
> If you use a custom exposure event make sure you send it when the user experiences the variant. More importantly, the custom exposure event may happen before assignment, meaning that the user property is not yet set, and the initial custom exposure event is not actually counted as an exposure in analysis. Note that it does not cost extra money to use Amplitude Exposure events.

I'm confused by this -- maybe it's just the ordering. 

- We are suggesting that the customer send the custom exposure event when the user experiences the variant. Is there a different option? Are people accidentally sending before they experience the variant? 
- What causes the custom exposure event to be sent before assignment? Is there anything the customer can do to prevent that so the initial custom exposure event is counted as an exposure in analysis?

Reply:
@bgiori Can you help clarify here thanks

Reply:
> We are suggesting that the customer send the custom exposure event when the user experiences the variant. Is there a different option? Are people accidentally sending before they experience the variant?

Yes, the custom exposure events are often existing events, e.g. `home page viewed`.

Since the order of assignment and exposure matters, if the custom event is ingested before the assignment event, the custom exposure event will not be counted.

Reply:
> What causes the custom exposure event to be sent before assignment? Is there anything the customer can do to prevent that so the initial custom exposure event is counted as an exposure in analysis?
 
 This is an intractable problem, and is the primary impetus behind created the "amplitude exposure event".

Reply:
I think the order for this paragraph should be:
1. Using custom exposure can cause SRM because...
2. Switching to amplitude exposure can help because....

rather than the other way around.